ooniprobe/Model/Database/Measurement.mm

```diff
@@ -50,6 +50,15 @@ -(NSString*)getLogFile{
     return [self.result_id getLogFile:self.test_name];
 }
 
+-(NSString*)getLocalizedStartTime{
+    //TODO we might need a copy for this
+    if (self.start_time == nil)
+        return NSLocalizedString(@"TestResults.UnknownASN", nil);
+    //from https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPInternational/InternationalizingLocaleData/InternationalizingLocaleData.html
+    NSString *localizedDateTime = [NSDateFormatter localizedStringFromDate:self.start_time dateStyle:NSDateFormatterShortStyle timeStyle:NSDateFormatterShortStyle];
```

Comment:
What happens if `self.start_time` is not a valid date (ex. it is the string `I am an invalid date`)?

Reply:
`start_time` can't be an invalid date.

Reply:
> `start_time` can't be an invalid date.

Where do you get this guarantee from? My understanding is that it is a string type so I believe it can be anything.

Reply:
it's an nsdate not a string

Reply:
Ah ok I understand now. Thanks for clarifying this.